WordPress/Classes/ViewRelated/Post/PostSettingsSelectionViewController.h

```diff
@@ -1,10 +1,18 @@
 #import <UIKit/UIKit.h>
 
+extern NSString * const SettingsSelectionTitleKey;
+extern NSString * const SettingsSelectionTitlesKey;
+extern NSString * const SettingsSelectionValuesKey;
+extern NSString * const SettingsSelectionDefaultValueKey;
+extern NSString * const SettingsSelectionCurrentValueKey;
+
 @interface PostSettingsSelectionViewController : UITableViewController
 
-@property (nonatomic, copy) void(^onItemSelected)(NSString *);
+@property (nonatomic, copy) void(^onItemSelected)(NSObject *);
```

Comment:
Just wondering here why we're avoiding the string?  Is this due to code that's coming up?